src/cider/nrepl/middleware/refresh.clj

```diff
@@ -76,12 +76,18 @@
    msg]
   (reload-utils/after-reply error msg))
 
+(def ^{:added "0.48.0"} user-indicated-clear?
```

Comment:
I don't think this name is good, as it's too specific and it's possible that some op might be triggered in a different way as opposed to a direct request from an IDE. (unlikely, but possible) Should this be named just `reset-in-progress` or something like this?